src/main/public/js/analytics-custom-event-sender.js

```diff
@@ -0,0 +1,69 @@
+$(function () {
+  function sendEvent(eventCategory, eventAction, eventLabel) {
+    if (!eventCategory) {
+      throw new Error('Event category is required')
+    }
+    if (!eventAction) {
+      throw new Error('Event action is required')
+    }
+    if (!eventLabel) {
+      throw new Error('Event label is required')
+    }
+
+    ga('send', 'event', eventCategory, eventAction, eventLabel)
+  }
+
+  function labelText(labelElement) {
```

Comment:
I wonder whether the label text should be going into the action, https://support.google.com/analytics/answer/1033068?hl=en

Real time view only goes back 30 minutes for me,
I found the other data in Behaviour => Events, but I can't find a way to tie the label back to an action.
And 'Not sure' is a bit too generic not to be able to tie it back.

Reply:
actually, its fine, managed to find the data